examples/cloudwatch-logs/variables.tf

```diff
@@ -85,7 +85,7 @@ variable "tags" {
 }
 
 variable "layer_arn" {
-  description = "In case you are using SSM This is the ARN of the Coralogix Security Layer."
+  description = "In case you are using SM This is the ARN of the Coralogix Security Layer."
```

Comment:
In case you are using secrets manager This is the ARN of the Coralogix Security Layer lambda.